packages/embed/src/index.jsx

```diff
@@ -0,0 +1,112 @@
+import React from "react"
+import { createRoot } from 'react-dom/client'
+import { Route, Routes, BrowserRouter } from 'react-router-dom'
+
+import './util/initWeb3'
+import App from './components/app'
+import {
+  HASH_ID_ROUTE,
+  ID_ROUTE,
+  COLLECTIBLES_ROUTE,
+  COLLECTIBLE_ID_ROUTE,
+  COLLECTIBLES_DISCORD_ROUTE,
+  COLLECTIBLE_ID_DISCORD_ROUTE,
+  AUDIO_NFT_PLAYLIST_DISCORD_ROUTE,
+  AUDIO_NFT_PLAYLIST_ROUTE
+} from './routes'
+
+import './index.css'
```

Comment:
Why remove the stems and harmony css imports in `index.css` but still import `index.css`?

Reply:
thx good catch